daily.sh

```diff
@@ -393,6 +393,7 @@ main () {
                                "purgeusers"
                                "bill_data"
                                "alert_log"
+                               "networks_purge"
```

Comment:
we really don't want to add more things to daily, it should go into the "new" scheduler instead.